src/SessionAccount.ts

```diff
@@ -86,6 +93,30 @@ export class SessionAccount {
     )
   }
 
+  private async signMessage(
+    outsideExecutionTypedData: TypedData,
+  ): Promise<Signature> {
+    const calls = (outsideExecutionTypedData.message as any).Calls.map(
```

Comment:
it might be nice to fail with with a more explicit message if we are not signing an EFO